### PR TITLE
Linked all of the pages as intended

### PR DESCRIPTION
**Pull request by arrianNaomi**

### CODE REVIEW

File: whyBeACS.html
Comment (JennaStilling) [inline whyBeACS.html:1]: We don't need class = "button" on the bottom nav bar. The button class is for selection buttons in other areas in the files. Nav bar ends up looking like this
image

File: careers-in-cs.html [hunk 1]
```diff
@@ -187,10 +187,14 @@ <h2>Mobile Application Developer</h2>
             </div>
         </div>
     </div>
-    <div class="bottomnav">
-        <a href="index.html">Home</a>
-        <a href="WhatIsItLike.html">Testemonies From Computer Scientists</a>
-    </div>
+    <div class="bottomnav"></div>
```
Comment (JennaStilling) [inline careers-in-cs.html]: Nav bar div tag is too high up, it ends before the links are wrapped
image

File: whyBeACS.html
Comment (JennaStilling) [reply whyBeACS.html:1]: For all bottom nav bars ^

**Review by JennaStilling — CHANGES_REQUESTED** [review]

File: careers-in-cs.html (hunk 1 above)
Comment (arrianNaomi) [reply careers-in-cs.html]: Updated

File: whyBeACS.html
Comment (arrianNaomi) [reply whyBeACS.html:1]: Updated

**Review by JennaStilling — APPROVED** [review]

**Pull request merged**